packages/components/pie-notification/src/index.ts

```diff
@@ -1,24 +1,193 @@
-import { LitElement, html, unsafeCSS } from 'lit';
-
-import { defineCustomElement } from '@justeattakeaway/pie-webc-core';
+import {
+    LitElement,
+    unsafeCSS,
+    TemplateResult,
+    nothing,
+} from 'lit';
+import { StaticValue, html, unsafeStatic } from 'lit/static-html.js';
+import { defineCustomElement, validPropertyValues } from '@justeattakeaway/pie-webc-core';
+import { property, query } from 'lit/decorators.js';
+import { type NotificationProps, variants, headingLevels } from './defs';
 import styles from './notification.scss?inline';
-import { NotificationProps } from './defs';
+
+import '@justeattakeaway/pie-icon-button';
+import '@justeattakeaway/pie-icons-webc/IconClose';
+import '@justeattakeaway/pie-icons-webc/IconInfoCircle';
+import '@justeattakeaway/pie-icons-webc/IconAlertCircle';
+import '@justeattakeaway/pie-icons-webc/IconAlertTriangle';
+import '@justeattakeaway/pie-icons-webc/IconCheckCircle';
 
 // Valid values available to consumers
 export * from './defs';
 
 const componentSelector = 'pie-notification';
+const componentClass = 'c-notification';
 
 /**
  * @tagname pie-notification
  */
 export class PieNotification extends LitElement implements NotificationProps {
-    render () {
-        return html`<h1 data-test-id="pie-notification">Hello world!</h1>`;
+    @property({ type: Boolean })
+    public isOpen = false;
+
+    @property()
+    @validPropertyValues(componentSelector, variants, 'neutral')
+    public variant: NotificationProps['variant'] = 'neutral';
+
+    @property({ type: Boolean })
+    public compact = false;
```

Comment:
We have a code convention of naming boolean properties such as this with an `is` prefixed. So in this case it would be `isCompact`. We've had a few discussions about this convention but it appears to have become the standard way across the library so I would suggest sticking to this format for consistency

Reply:
I agree! Thanks!